backend/coins/btc/handlers/handlers.go

```diff
@@ -381,15 +391,26 @@ func (handlers *Handlers) getUTXOs(*http.Request) (interface{}, error) {
 }
 
 func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error) {
+	type result struct {
+		Success   bool                   `json:"success"`
+		ErrorCode string                 `json:"errorCode,omitempty"`
+		Balance   map[string]interface{} `json:"balance,omitempty"`
+	}
 	balance, err := handlers.account.Balance()
 	if err != nil {
-		return nil, err
+		return result{
+			Success:   false,
+			ErrorCode: err.Error(),
```

Comment:
The error is not always an error code (e.g. if it's coming from `transactions.Balance()`)? You should only put it here if you identify it as being a code.